### PR TITLE
[ES] Add `Score`

### PR DESCRIPTION
**Pull request by Ekkusu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Score/es.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Para más detalles de cómo funciona el ScoreV1 en cada modo de juego, consulte:
+
+- [osu!](ScoreV1/osu!)
+- [osu!taiko](ScoreV1/osu!taiko)
+- [osu!catch](ScoreV1/osu!catch)
+- [osu!mania](ScoreV1/osu!mania)
+
+## ScoreV2
+
+El ScoreV2 es una nueva iteración del sistema de puntuación. La idea principal detrás de esto es la estandarización de los sistemas de puntuación de todos los modos de juego, de modo que se otorga una puntuación perfecta de 1,000,000 de puntos con un modificar de puntuación de 1.00x, con ganancias adicionales de puntuación además de las bonificaciones de las ruletas de osu!, los redobles de tambor de osu!taiko, y las bananas de osu!catch. Esto implica en una desviación de los valores de puntuación originales de cada objeto de golpeo, y un cambio al sistema que se centra más en las proporciones y escalas al límite de 1 millón.
```
Comment (agatemosu) [inline wiki/Gameplay/Score/es.md:24]:
```suggestion
El ScoreV2 es una nueva iteración del sistema de puntuación. La idea principal detrás de esto es la estandarización de los sistemas de puntuación de todos los modos de juego, de modo que se otorga una puntuación perfecta de 1,000,000 puntos con un modificador de puntuación de 1.00x, con ganancias adicionales de puntuación además de las bonificaciones de las ruletas de osu!, los redobles de tambor de osu!taiko, y las bananas de osu!catch. Esto implica en una desviación de los valores de puntuación originales de cada objeto de golpeo, y un cambio al sistema que se centra más en las proporciones y escalas al límite de 1 millón.
```

Comment (Ekkusu) [reply wiki/Gameplay/Score/es.md:24]: - En la parte de eliminar el "de" no me convence, pero lo tengo en cuenta. Mientras tanto, seguiré buscando su forma correcta de escribirlo.
- En modificar el verbo "modificar", muchas gracias por mostrarme el error que se me paso de alto, y ya lo corrijo.
- Y como pregunta, mientras investigaba, me encontré con este articulo de la RAE (https://www.rae.es/dpd/n%C3%BAmeros#:~:text=a)%20Al,8%20327%20/%20451), en el cual no se si debería aplicarlo en este texto...   :/

Comment (agatemosu) [reply wiki/Gameplay/Score/es.md]: - Sobre quitar el "de" tienes razón, no se debería quitar.
- En el `Article_styling_criteria/Writing` pone que se debe usar el [Manual de estilo de Wikipedia en inglés sobre el formato de números](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Numbers)

File: wiki/Gameplay/Score/es.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Puntuación 
+
+*Para otros usos, consulte: [puntuación (desambiguación)](/wiki/Disambiguation/Score).*
+
+La puntuación que recibe un jugador después de completar un [mapa](/wiki/Beatmap) esta determinado por los [juicios](/wiki/Gameplay/Judgement) que el jugador recibió por cada [objetos de golpeo](/wiki/Gameplay/Hit_object). El sistema de puntuación se divide en dos versiones principales que actualmente está en uso.
```
Comment (agatemosu) [inline wiki/Gameplay/Score/es.md:5]:
```suggestion
La puntuación que recibe un jugador después de completar un [mapa](/wiki/Beatmap) está determinado por los [juicios](/wiki/Gameplay/Judgement) que el jugador recibió por cada [objeto de golpeo](/wiki/Gameplay/Hit_object). El sistema de puntuación se divide en dos versiones principales que actualmente está en uso.
```

File: wiki/Gameplay/Score/es.md (hunk 1 above)
Comment (Ekkusu) [reply wiki/Gameplay/Score/es.md]: Bien, entonces dejare el problema como solucionado y muchas gracias por tu aporte :)

**Review by agatemosu — APPROVED** [review]

**Review by SrIzan10 — APPROVED** [review]
looks good 👍

**Comment by SrIzan10** [comment]
you can check both boxes now

**Comment by SrIzan10** [comment]
nvm tengo cosas que decir

**Comment by Ekkusu** [comment]
> nvm tengo cosas que decir

Puede decirlo libremente

**Comment by Ekkusu** [comment]
> you can check both boxes now by editing your message

I would think that this part is handled by an administrator

**Comment by TicClick** [comment]
the first one is for pull request authors to set up, it indicates that they have checked the PR according to a short linked guide. the second may be set by whomever gets to it first (but it's usually the pull request author again who knows if a reviewer is proficient in a given language or not)

File: wiki/Gameplay/Score/es.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Puntuación 
+
+*Para otros usos, consulte: [puntuación (desambiguación)](/wiki/Disambiguation/Score).*
```
Comment (TicClick) [inline wiki/Gameplay/Score/es.md:3]: in notes of all sorts (`For ..., see ...`, `See also: ...`, and so on), article names are usually capitalized -- shouldn't that be the case even in Spanish?
```suggestion
*Para otros usos, consulte: [Puntuación (desambiguación)](/wiki/Disambiguation/Score).*
```

Comment (agatemosu) [reply wiki/Gameplay/Score/es.md:3]: Yes, it should be capitalized

File: wiki/Gameplay/Score/es.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Además de la estandarización mejorada, el ScoreV2 también es una solución mejorada para un problema de [desbordamientos aritmético](https://es.wikipedia.org/wiki/Desbordamiento_aritm%C3%A9tico) que puede surgir en mapas largos y con combos grandes. Debido a que la puntuación total se almacena como un numero entero de 32-bit y teóricamente el ScoreV1 puede dar una cantidad ilimitada de puntuación, pero exceder el máximo entero representable de los 32-bit siendo el valor máximo de 2,147,483,647 puntos. Haría que el contador de puntuación se ajustara a valores negativos (lo cual visualmente se ve como si la puntuación contara hacia atrás). En la práctica, el ScoreV2 se habilita automáticamente para puntajes establecidos en mapas de tiempo largos que, de otro modo, tendrían un máximo puntaje por encima del límite de enteros de 32-bit.
```
Comment (agatemosu) [inline wiki/Gameplay/Score/es.md:26]:
```suggestion
Además de la estandarización mejorada, el ScoreV2 también es una solución mejorada para un problema de [desbordamientos aritmético](https://es.wikipedia.org/wiki/Desbordamiento_aritm%C3%A9tico) que puede surgir en mapas largos y con combos grandes. Debido a que la puntuación total se almacena como un número entero de 32-bit y teóricamente el ScoreV1 puede dar una cantidad ilimitada de puntuación, pero exceder el máximo entero representable de los 32-bit siendo el valor máximo de 2,147,483,647 puntos. Haría que el contador de puntuación se ajustara a valores negativos (lo cual visualmente se ve como si la puntuación contara hacia atrás). En la práctica, el ScoreV2 se habilita automáticamente para puntajes establecidos en mapas de tiempo largos que, de otro modo, tendrían un máximo puntaje por encima del límite de enteros de 32-bit.
```

**Review by TicClick — APPROVED** [review]

**Comment by SrIzan10** [comment]
> the first one is for pull request authors to set up, it indicates that they have checked the PR according to a short linked guide. the second may be set by whomever gets to it first (but it's usually the pull request author again who knows if a reviewer is proficient in a given language or not)

yeah, I guess I'm proficient as I'm spanish and also I'll start reviewing PRs more often

**Pull request merged**